src/components/ReserveButton.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { StyledButton } from './styles/ReserveButton.styled'
+
+type Props = {
+  filled: boolean
+  buttonText: string
+}
+
+const ReserveButton = (props: Props) => {
+  return <StyledButton filled={props.filled}>{props.buttonText}</StyledButton>
+}
```

Comment:
Looks exactly like the other Button component. Duplicate code for the same function.

You can make both of them the same, but instead of passing in just 2 props, you can add in one more, `type: RESERVE | PRIMARY`